rct229/rulesets/ashrae9012019/section22/section22rule36.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
             each_rule=Section22Rule36.PrimaryFluidlLoop(),
             index_rmd=BASELINE_0,
             id="22-36",
-            description="Baseline chilled water system that does not use purchased chilled water shall be modeled with constant flow primary loop and variable flow secondary loop.",
+            description="Baseline chilled water systems that do not use purchased chilled water shall be modeled with constant flow primary loop and variable flow secondary loop.",
```

Comment:
Baseline chilled water systems that do not use purchased chilled water shall be modeled with a constant flow primary loop and a variable flow secondary loop.